### PR TITLE
Add SentryShot

### PR DESCRIPTION
**Pull request by Curid**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [ ] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [ ] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

Docker is optional not a requirement, should I remove it?

The project is over 2 years old but it was just re-released under a new name and repo.

### CODE REVIEW

File: software/sentryshot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "SentryShot"
+website_url: "https://github.com/SentryShot/sentryshot"
+source_code_url: "https://codeberg.org/SentryShot/sentryshot"
+description: "Video Management System"
```
Comment (KieranRobson) [inline software/sentryshot.yml:4]:
```suggestion
description: "Video Management System."
```

File: software/sentryshot.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: "SentryShot"
+website_url: "https://github.com/SentryShot/sentryshot"
+source_code_url: "https://codeberg.org/SentryShot/sentryshot"
```
Comment (KieranRobson) [inline software/sentryshot.yml:3]: In my opinion if the github is just a mirror, the codeberg should always be used instead.

```suggestion
website_url: "https://codeberg.org/SentryShot/sentryshot"
source_code_url: "https://codeberg.org/SentryShot/sentryshot"
```

File: software/sentryshot.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+description: "Video Management System"
+licenses:
+  - GPL-2.0
+platforms:
+  - Rust
+  - Docker
```
Comment (KieranRobson) [inline software/sentryshot.yml:9]: Just a quick change in order to match the docs

```suggestion
platforms:
  - Docker
  - Rust
```

File: software/sentryshot.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+tags:
+  - Video Surveillance
+depends_3rdparty: no
```
Comment (KieranRobson) [inline software/sentryshot.yml:12]: This is optional so can be removed.

```suggestion
  - Video Surveillance
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple easy changes to be made.

File: software/sentryshot.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+name: "SentryShot"
+website_url: "https://codeberg.org/SentryShot/sentryshot"
+source_code_url: "https://codeberg.org/SentryShot/sentryshot"
+description: "Video Management System."
```
Comment (nodiscc) [inline software/sentryshot.yml:4]:
```suggestion
description: "Video surveillance management system."
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**